### ISSUE TITLE
Polychromatic doesn't launch on openSUSE Tumbleweed since version 0.7.0

### ISSUE THREAD

**Issue by DarkWav** [polychromatic/polychromatic#341]
### Details about the problem

polychromatic-controller and polychromatic-tray-applet crash on startup on openSUSE Tumbleweed since version 0.7.0.
Terminal output:
```
polychromatic-controller:
Initialising application...
Initialising window...
Traceback (most recent call last):
  File "/usr/bin/polychromatic-controller", line 588, in <module>
    app.load()
  File "/usr/bin/polychromatic-controller", line 121, in load
    self.main_window = MainWindow()
  File "/usr/bin/polychromatic-controller", line 216, in __init__
    if os.path.getsize(os.path.join(common.paths.data_dir, "qt", "style.qss")) < 100:
  File "/usr/lib64/python3.8/genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] File or Directory not found: '/usr/lib/python3.8/site-packages/data/qt/style.qss'

polychromatic-tray-applet:
Tray icon missing: img/tray/light/breeze.svg
Using fallback icon.                                                                                                         
Traceback (most recent call last):
  File "/usr/bin/polychromatic-tray-applet", line 954, in <module>
    app.start()
  File "/usr/bin/polychromatic-tray-applet", line 262, in start
    self.build_indicator()
  File "/usr/bin/polychromatic-tray-applet", line 316, in build_indicator
    indicator.setup(self._get_tray_icon())
  File "/usr/bin/polychromatic-tray-applet", line 120, in setup
    self.indicator = appindicator.Indicator.new("polychromatic-tray-applet", icon_path, appindicator.IndicatorCategory.APPLICATION_STATUS)
TypeError: Argument 1 does not allow None as a value
```
The crash indicates missing Qt libraries, however, I was unable to diagnose which ones.
The crash is not reproducible within Fedora 34 KDE, everything works fine there.
OpenRazer itself works fine, it is fully functional with e.g. RazerGenie.

### Operating System / Desktop Environment / Version(s)

openSUSE Tumbleweed 20210807
KDE Plasma 5.22.4
KDE Frameworks 5.84.0
Qt 5.15.2
Kernel 5.13.8

### Affected Devices

All devices (I'm using a Razer Mamba Wireless (2018) at the moment, but polychromatic doesn't even start.)

**Comment by lah7** [comment]
I've fixed the cause of the issue in 5c6c5d7ad181efeb912fdb80b7657846ba0a82ec. If deleting `/usr/lib/python3.8/site-packages/data` isn't an option, [patch `/usr/lib/python3.8/site-packages/polychromatic/common.py`](https://github.com/polychromatic/polychromatic/commit/5c6c5d7ad181efeb912fdb80b7657846ba0a82ec.diff) with the fix.

**Comment by DarkWav** [comment]
Thanks for the info!
I just found out that somehow the folder `/usr/lib/python3.8/site-packages/data` existed as an empty folder on my system for unknown reasons, deleting it fixed the issue permanently.
Thank you for the fix, issue solved.
